### PR TITLE
Main fragment/ overflow menu text style (EXPOSUREAPP-12611)

### PR DESCRIPTION
**Pull request by SamuraiKek**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12611)

Aligns overflow menu text spacing with design specs.

### CODE REVIEW

**Review by schauersbergern — COMMENTED** [review]
Should't this be applied to all the menus? See "Additional Remarks" in Ticket?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/MenuAccessibilityExtensions.kt [hunk 1]
```diff
@@ -18,3 +22,11 @@ fun Menu.setItemContentDescription(context: Context) {
         }
     }
 }
+
+fun Menu.setTextSize() {
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/MenuAccessibilityExtensions.kt:26]: can you please a common style for the whole App, See `Theming menus` under
https://m3.material.io/components/menus/implementation

**Comment by SamuraiKek** [comment]
> Should't this be applied to all the menus? See "Additional Remarks" in Ticket?

Ah, I missed that part. Yes, I'll try to apply it to  all of them.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/MenuAccessibilityExtensions.kt (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/MenuAccessibilityExtensions.kt:26]: Thanks, I'll have a look. I did try to go this route first, but the answers I found were saying to include this line in my theme `app:itemTextAppearance="@style/menu_text_style"` but it didn't have any effect.

**Review by mtwalli — APPROVED** [review]
Lgtm

**Review by schauersbergern — APPROVED** [review]

**Pull request merged**